CODING-GUIDELINES.md

```diff
@@ -0,0 +1,23 @@
+## Coding Style Guidelines
+
+The purpose of this document is to set forth standards for current and future contributions to Joplin's source code to allow for easier management of resources, quicker bug fixes, and an improved overall experience.
+
+
+## Coding Style
+
+
+- All functions, classes, components and properties must be clearly and concisely named so that upon inspection, one can quickly and easily discern what the purposes of each are and how they are associated with the project as a whole.
+- In order to further improve code readability, comments are welcome but need to be as short and precise as needed in order to assist but not distract from the rest of the code.
+
+## Desktop (ElectronClient) guidelines
+
+
+- For changes that affect the user interface, refer to `ElectronClient/app/theme.js` for all styling information. This is to allow for easy navigation of Joplin's various features and improve the overall user experience.
```

Comment:
Please remove this file from the commit. Coding guidelines would definitely be useful, but there's currently not much time to look into that due to GSoC.